spec/adapters/publishing_adapter_spec.rb

```diff
@@ -537,5 +537,14 @@
 
       subject.redirect_section(section, to: "/new-location")
     end
+
+    it "redirects section via Publishing API with valid attributes" do
+      expect(publishing_api).to receive(:put_content).with(
+        redirect_content_id,
+        be_valid_against_schema("redirect")
```

Comment:
This reads odd to me compared to the version it was moved from. Does `be_valid_against_schema` return the JSON (which happens to be truthy and therefore works as an expectation in the original spec)? I think it would be useful to split those two concerns into two separate expectations.